internal/common/baseclient/baseclient.go

```diff
@@ -130,9 +130,11 @@ func (c *BaseClient) Logf(format string, a ...interface{}) {
 
 // GetVoteForRule returns the client's vote in favour of or against a rule.
 // COMPULSORY: vote to represent your island's opinion on a rule
-func (c *BaseClient) GetVoteForRule(ruleName string) bool {
+func (c *BaseClient) VoteForRule(ruleName string) (bool, bool) {
 	// TODO implement decision on voting that considers the rule
-	return true
+	approve := true
+	abstain := false
+	return approve, abstain
```

Comment:
I wouldn't go with 2 bools since it leads to invalid combinations (such as approve = true, abstain = true)
I would instead make a VoteResult enum that can take on the values `Approve`, `Reject`, `Abstain` and return a value of this enum

Reply:
done! thanks!